packages/accordions/src/elements/accordion/Accordion.tsx

```diff
@@ -39,11 +39,15 @@ interface IAccordionProps extends Omit<HTMLAttributes<HTMLDivElement>, 'onChange
   isBare?: boolean;
   /** Determines if panels can be collapsed in an uncontrolled accordion */
   isCollapsible?: boolean;
-  /** Reduces padding on accordion headers and panels */
+  /** Reduces the padding on the accordion headers and panels */
   isCompact?: boolean;
-  /** Determines if multiple panels can be expanded at the same time in an uncontrolled accordion */
+  /** Determines if multiple panels can be expanded simultaneously in an uncontrolled accordion*/
   isExpandable?: boolean;
-  /** A callback that is invoked with an accordion section’s index when the expanded state changes */
+  /**
+   * Handles accordion state expansion changes
+   *
+   * @param {number} index The index being expanded
```

Comment:
Let's be more specific, something like:

`@param {number} The index of the section being expanded`

Reply:
Fixed in the latest update